routers/planRouter.js

```diff
@@ -2,8 +2,10 @@ const { Router } = require('express');
 const planController = require('../controllers/planController');
 const planRouter = Router();
 
-planRouter.get('/', planController.getAllPlans)
-planRouter.post('/', planController.purchasePlan)
+planRouter.get('/', planController.getAllPlans);
+planRouter.get('/', planController.getPlanByName);
+
+// planRouter.post('/', planController.purchasePlan);
```

Comment:
do not commit commnented out lines

Reply:
We will use line 8 in the future for handling buying plans